src/fetch-api/ExtensionInstallations.js

```diff
@@ -0,0 +1,59 @@
+import _ from 'lodash';
+import { fetchResource } from '../resources/Resource';
+import extensionInstallationResource, { INSTALLATIONS } from '../resources/extensionInstallations';
+
+export default class ExtensionInstallations {
```

Comment:
Please document this class. Add a description that explains what is an extension installation.